plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -79,4 +82,22 @@ public void testShowCreateTable()
                         format("   location = '.*/" + schemaName + "/region'\n") +
                         "\\)");
     }
+
+    @Test
+    public void testHiddenPathColumn()
```

Comment:
@findinpath added this test, borrowed from https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java#L4135